playbooks/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -54,7 +54,11 @@ openshift_openstack_external_network_name: "public"
 
 # # Kuryr can use a different subnet per namespace
 # openshift_kuryr_subnet_driver: namespace
+# # Kuryr can use different security group driver to provide different
+# isolation policies. To enable the namespace isolation, uncomment the next:
 # openshift_kuryr_sg_driver: namespace
+# And to make use of the network policies uncomment the next:
+# openshift_kuryr_sg_driver: policy
```

Comment:
This is slightly unclear; I assume that openshift_kuryr_sg_driver should either be set to namespace or policy, correct? This could be misinterpreted as saying that there should be two openshift_kuryr_sg_driver entries.

Reply:
Ok, I'll try to make it clear. Thanks!